src/triage/component/results_schema/nuke_triage.sql

```diff
@@ -0,0 +1,48 @@
+/*
+  Function for wiping all triage tables, schemas and indexes.
+Useful  when ou try to start clean (again) and try to recover some
+previous errors
+ */
+create or replace function nuke_triage()
+    returns text as $result$
+
+    declare
+    result text;
+    query text;
+
+    begin
```

Comment:
should we include some sort of tests to ensue this file remains updated if the triage schema changes?

Reply:
I need to check how to test `sql` in a nice way. Will open a issue.

Reply:
Opened issue #802